### ISSUE TITLE
Allow user to override existing theme

### ISSUE THREAD

**Issue by malliaridis** [malliaridis/penpot-plugin-material-theme-builder#11]
### Description

Since the user may want to create and update the theme, the plugin should support updating existing themes if present.

### Acceptance Criteria

- Existing theme colors are updated
- Design elements automatically update to the new theme

### Additional Information

Deprecated values that may be removed because no longer in use will be addressed in a separate issue.

This issue is related to #19, since the theme name may be used for identifying existing themes.

This issue is blocked by #20, as the solution for this issue should also update the color value in the input field of the color.
